src/main/java/AuthenticationProject/Services/UserService.java

```diff
@@ -15,13 +19,18 @@ public static synchronized UserService getInstance()
         return userService;
     }
 
-    public void updateEmail(String email) {
+    public void updateEmail(int id, String email) {
+        UserRepository.updateEmail(int id, String email);
```

Comment:
you shold call the userRepository function like this: UserRepository.updateEmail(id, email);
with the parameters you got